lib/galaxy_test/api/test_workflows.py

```diff
@@ -228,7 +229,7 @@ def test_assign_column_pja(self):
 # - Allow post to workflows/<workflow_id>/run in addition to posting to
 #    /workflows with id in payload.
 # - Much more testing obviously, always more testing.
-class WorkflowsApiTestCase(BaseWorkflowsApiTestCase, ChangeDatatypeTestCase):
+class WorkflowsApiTestCase(BaseWorkflowsApiTestCase, ChangeDatatypeTestCase, UsesShed):
```

Comment:
If you're using `UsesShed` you need to make it an integration test case, but I don't think you need to install a shed tool, you could just specify an invalid tool id

Reply:
sorry, I meant tool version